install.sh

```diff
@@ -52,6 +52,42 @@ main() {
     install_from_archive
 }
 
+confirm_install_options() {
+    local _version="$1"
```

Comment:
As you may see the ongoing dicussion here #852, we have changed a little bit the installation script. We are going to simplify this and follow elasticsearch `tar.gz` install.

Reply:
Let's wait until tomorrow before updating the script.